VideoLocker/src/main/java/org/edx/mobile/http/Api.java

```diff
@@ -223,6 +224,18 @@ public ProfileModel getProfile() throws Exception {
         // hold the json string as it is
         res.json = json;
 
+
+        // store profile json
+        if (json != null ) {
```

Comment:
@hanningni makes sense to me. Maybe for things like this we should leave a TODO comment or something?